lxd/device/nic_routed.go

```diff
@@ -110,6 +112,33 @@ func (d *nicRouted) validateConfig(instConf instance.ConfigReader) error {
 		}
 	}
 
+	for _, keyPrefix := range []string{"ipv4", "ipv6"} {
```

Comment:
I don't think we should be detecting duplicate addresses on the network during validation stage, but rather on instance device start. As otherwise this will cause configuration delays, and may cause issues if you are just configuring a device with an address that is in use right now, but may not be at instance start time.

Reply:
I've moved this logic to `nicRouted.Start` and updated the integration test.